Cargo.toml

```diff
@@ -48,7 +49,8 @@ v2_58 = ["v2_56", "glib-sys/v2_58", "gobject-sys/v2_58"]
 v2_60 = ["v2_58", "glib-sys/v2_60"]
 v2_62 = ["v2_60", "glib-sys/v2_62", "gobject-sys/v2_62"]
 v2_64 = ["v2_62", "glib-sys/v2_64"]
-dox = ["glib-sys/dox", "gobject-sys/dox"]
+log_macros = ["log"]
```

Comment:
Can you change `.travis.yml` to to also enable both new features in the cases it builds for the latest GLib version currently?

Reply:
I can do it, but looking at `.travis.yml` it seems that it does two builds, one against the empty feature set, and the other against `v2_48`.

Do you want me to add `log_macros` beside `v2_48` or upgrade to the latest, while we are at it (considering that structured logging requires `v2_56`)?

Reply:
Add it besides the other one. I don't think we can update to latest because travis runs an ancient version of Ubuntu, but maybe that changed in the meantime.